hail/src/main/scala/is/hail/rvd/OrderedRVD.scala

```diff
@@ -571,10 +571,10 @@ class OrderedRVD(
   }
 
   def forall(p: RegionValue => Boolean): Boolean =
-    crdd.map(p).run.forall(x => x)
+    crdd.map(p).clearingRun.forall(x => x)
```

Comment:
were these memory bugs?

Reply:
They look like memory leaks to me. I ran it past @danking and he agreed.